cfme/tests/infrastructure/test_snapshot.py

```diff
@@ -463,6 +463,44 @@ def test_snapshot_history_btn(create_vm, provider):
     assert snapshot_view.is_displayed
 
 
+@pytest.fixture
+def add2snaps_del1(provider, create_vm):
+    has_name = not provider.one_of(RHEVMProvider)
+    snapshot1 = new_snapshot(create_vm, has_name=has_name)
+    snapshot1.create()
```

Comment:
Required:
Need teardown `deletion` for this snapshot

Reply:
@prichard77 correct me if I'm wrong, but I think because the `create_vm` will be cleaned up all the remaining snapshots will too.

Reply:
That is what I was thinking @john-dupuy .

Reply:
ahhh understand function scope vm :+1: